de/privacy/current.md

```diff
@@ -275,7 +272,7 @@ Der Versand unseres Newsletters erfolgt über MailChimp, einer E-Mail Versandpla
 
 Die im Rahmen der Registrierung erfassten Daten, also Deine E-Mail Adresse, Deinen Nach- und Vornamen Daten über Deine Interaktion mit dem Newsletter IP-Adresse, Browsertyp und Betriebssystem
 
-Diese Daten werden in der Regeln auf Servern von Mailchimp in den USA gespeichert.
+Mailchimp verwendet für die Datenübertragung in die USA die [Standardvertragsklauseln (SCC)](https://mailchimp.com/legal/data-processing-addendum/) als rechtliche Grundlage, um den Schutz deiner Daten sicherzustellen. Die Daten werden auf Servern in den USA gespeichert.
```

Comment:
```suggestion
Mailchimp verwendet für die Datenübertragung in die USA die [Standardvertragsklauseln (SCC)](https://mailchimp.com/legal/data-processing-addendum/) als rechtliche Grundlage, um den Schutz deiner Daten sicherzustellen.
```
We have already said the data can go to USA, we don't need to repeat it.

By the way, Mailchimp is not written in PascalCase as we can read in their articles (Mailchimp, not MailChimp). We could change all other places, although it is not important.